core/src/main/scala/clue/ApolloClient.scala

```diff
@@ -76,14 +75,14 @@ protected object State {
   ) extends State[F, CP](PersistentClientStatus.Connecting)
 }
 
-class ApolloClient[F[_], S, CP, CE](
-  uri:                  Uri,
+class ApolloClient[F[_], P, S, CP, CE](
```

Comment:
Question: is the apollo protocol independent of websockets? Since `ApolloWebSocketClient` is a specific interface.

Reply:
Actually, I don't think it is. The successor spec, "enisdenjo" is for websockets too. The protocol could still be used over other layers. Do you think it's worth it to drop this abstraction? It would mostly remove the `CP` and `CE` parameters.

Reply:
Thanks for clarifying that! Yeah, then I don't think it's worth keeping the abstraction, especially if we can vanish a couple type parameters.

Reply:
Ok, I'll address that in a followup PR. There are changes in explore waiting for this PR and I don't want to hold them up.